File: February 17, 2026.js
function checkEligibility(athleteWeights, sledWeight) {

  const person = athleteWeights.length;
  const totalWeight = athleteWeights.reduce((sum, n) => sum + n, 0) + sledWeight;
  
  if ((sledWeight < 162 || totalWeight > 247) && (person == 1)) {
    return "Not Eligible"
  } else if ((sledWeight < 170 || totalWeight > 390) && (person == 2)) {
    return "Not Eligible"
  } else if ((sledWeight < 210 || totalWeight > 630) && (person == 4)) {
    return "Not Eligible"
  }

  return "Eligible"
}

File: February 17, 2026.py
def check_eligibility(athlete_weights, sled_weight):

    person = len(athlete_weights)
    total_weight = sum(athlete_weights) + sled_weight


    if ((sled_weight < 162 or total_weight > 247) and person == 1):
        return "Not Eligible"
    elif ((sled_weight < 170 or total_weight > 390) and person == 2):
        return "Not Eligible"
    elif ((sled_weight < 210 or total_weight > 630) and person == 4):
        return "Not Eligible"

    return "Eligible"